core/src/main/scala/org/scalarelational/instruction/Query.scala

```diff
@@ -55,4 +59,43 @@ case class Query[Expressions, Result](expressions: Expressions,
   def async = table.datastore.async {
     result
   }
+
+  def to[R](implicit manifest: Manifest[R]) = {
```

Comment:
I was thinking about that too, but then we would need to distinguish between ``InsertSingle``, ``InsertMultiple``, ``MappedInsertSingle``, ``MappedInsertMultiple``, ``Update`` and ``MappedUpdate``. This has some ramifications on the overall architecture (like a lot of added bloat in ``SQLContainer``).